openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/TestSummary.test.tsx

```diff
@@ -0,0 +1,156 @@
+};
+const mockHistory = {
+  push: jest.fn(),
+  goBack: jest.fn(),
+};
+
+jest.mock('rest/testAPI', () => {
+  return {
+    getListTestCaseResults: jest
+      .fn()
+      .mockImplementation(() =>
+        Promise.resolve({ data: MOCK_TEST_CASE_RESULT })
+      ),
+  };
+});
+jest.mock('react-router-dom', () => {
+  return {
+    ...jest.requireActual('react-router-dom'),
+    useHistory: jest.fn().mockImplementation(() => mockHistory),
+  };
+});
+
+jest.mock('components/DatePickerMenu/DatePickerMenu.component', () => {
+  return jest
+    .fn()
+    .mockImplementation(() => <div>DatePickerMenu.component</div>);
+});
+jest.mock('../../common/error-with-placeholder/ErrorPlaceHolder', () => {
+  return jest
+    .fn()
+    .mockImplementation(() => <div>ErrorPlaceHolder.component</div>);
+});
+jest.mock('../../Loader/Loader', () => {
+  return jest.fn().mockImplementation(() => <div>Loader.component</div>);
+});
+jest.mock('../../schema-editor/SchemaEditor', () => {
+  return jest.fn().mockImplementation(() => <div>SchemaEditor.component</div>);
+});
+
+describe('TestSummary component', () => {
+  it('Component should render', async () => {
+    render(<TestSummary {...mockProps} />);
+
+    const graphContainer = await screen.findByTestId('graph-container');
+    const graph = queryByAttribute(
+      'id',
+      graphContainer,
+      `${mockProps.data.name}_graph`
+    );
+
+    expect(
+      await screen.findByTestId('test-summary-container')
+    ).toBeInTheDocument();
+    expect(
+      await screen.findByTestId('test-case-expand-button')
+    ).toBeInTheDocument();
+    expect(await screen.findByTestId('params-container')).toBeInTheDocument();
+    expect(graphContainer).toBeInTheDocument();
+    expect(graph).toBeInTheDocument();
+    expect(
+      await screen.findByText('DatePickerMenu.component')
+    ).toBeInTheDocument();
+    expect(await screen.findAllByTestId('parameter-value')).toHaveLength(
+      mockProps.data.parameterValues?.length || 0
+    );
+  });
+
+  it('Show no data placeholder when there is no result, other CTA should also visible', async () => {
+    (getListTestCaseResults as jest.Mock).mockImplementationOnce(() =>
+      Promise.resolve({ data: [] })
+    );
+    render(<TestSummary {...mockProps} />);
+    const graphContainer = await screen.findByTestId('graph-container');
+    const graph = queryByAttribute(
+      'id',
+      graphContainer,
+      `${mockProps.data.name}_graph`
+    );
+
+    expect(graphContainer).toBeInTheDocument();
+    expect(graph).not.toBeInTheDocument();
+    expect(
+      await screen.findByText('ErrorPlaceHolder.component')
+    ).toBeInTheDocument();
+    expect(
+      await screen.findByTestId('test-case-expand-button')
+    ).toBeInTheDocument();
+    expect(await screen.findByTestId('params-container')).toBeInTheDocument();
+
+    expect(await screen.findAllByTestId('parameter-value')).toHaveLength(
+      mockProps.data.parameterValues?.length || 0
+    );
+  });
```

Comment:
If there is no data, this should not be visible right?

Reply:
No, it should visible if there is no data for results.